examples/logging_example2.py

```diff
@@ -31,18 +31,21 @@
     'STA1_x': 5.0,
     'stepsTime': 1,
     'minPower': 0,
-    'steps': 2,
```

Comment:
is this necessary?

Reply:
This is just to ensure that the example does not take a very long time to run. The default `steps` value in the ns-3 example is set to `200`. According to me, that might take quite a long time to run. What do you guys think?

Reply:
sorry for not being clear, i meant: why did this change from `2` to `[2]`? It's the only element set as a list.
If it's possible to make this dictionary uniform (either all lists or all values) i'd go for it